index.html

```diff
@@ -50,6 +58,190 @@ <h1>Hey there.
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div>
+
+                <div class="cards">
+                    <h2 class="cardtitle">Profesional Art Printing Data</h2>
+                    <p class="cardsupporting">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="tags catags">
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div>
+
+                <div class="cards">
+                    <h2 class="cardtitle">Profesional Art Printing Data</h2>
+                    <p class="cardsupporting">
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. 
+                        has been the industry's standard
+                    </p>
+                    <ul class="tags catags">
+                        <li class="taging cataging"><p class="tag catag">html</p></li>
+                        <li class="taging cataging"><p class="tag catag">bootstrap</p></li>
+                        <li class="taging cataging"><p class="tag catag">Ruby</p></li>
+                    </ul>
+                    <div class="buttn">
+                        <button type="button">See Project</button> 
+                    </div> 
+                </div>         
+            </section>
+        </section>
+
+        <section class="aboutme">
+
+            <div class="about">
+                <div class="primary">
+                    About me
+                </div>
+                <div class="secondary">
+                    <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div class="buttona"><button type="button">Get My Resume</button> </div>
```

Comment:
- [ ] [Optional] You have implemented the `hover` effect but the `pressed` effect is not working. Kindly implement it
### Design
![ff](https://user-images.githubusercontent.com/81727988/177008691-547e343a-e3a3-4bc4-a6e2-0d7950d2edca.png)

Reply:
thanks @roshan-bajgain, sure will do